build.gradle

```diff
@@ -11,7 +11,7 @@ compileJava {
 }
 
 // SkBee version
-version = '3.6.1'
+version = 'INSERT VERSION'
```

Comment:
Let back out these changes until the final dependencies are locked in place.

Reply:
On that note, SkBee nor Skript has ever used "INSERT VERSION" here.
that is only used when creating new syntaxes and is used in `@Since("")`